### DISCUSSION TITLE
List dates ?

### DISCUSSION THREAD

**Discussion by KaKi87** [ICIJ/datashare#958] [Q&A]
Hello,

Datashare allows listing persons, organizations and locations inside a document.

How about dates, durations, etc. ?

Thanks

**Comment by pirhoo** [comment]
Hi @KaKi87, 

it doesn't support extracting dates or duration yet but we would be more than happy to review any extension to add such feature:

https://github.com/ICIJ/datashare/wiki/Backend-%E2%80%BA-Extensions
https://github.com/ICIJ/?q=datashare-extension-
